lib/vanagon/platform/windows.rb

```diff
@@ -170,7 +162,7 @@ def output_dir(target_repo = "")
       #
       # @return [String] the cygwin drive root
       def drive_root
-        "/cygdrive/c"
+        "C:"
```

Comment:
We need cygpath to get rid shortcuts (`Progra~1`) and to specify `PATH` in the cygwin build environment